mac_cleanup/core.py

```diff
@@ -192,8 +192,8 @@ def __filter_modules(module_: BaseModule, filter_type: Type[T]) -> TypeGuard[T]:
 
         return isinstance(module_, filter_type)
 
-    def _count_dry(self) -> float:
-        """Counts free space for dry run :return: Approx amount of bytes to be removed."""
+    def extract_paths(self) -> (Path_, float):
+        """Extracts all paths from the collector :return: List of paths with size."""
```

Comment:
just to mark it somehow as a generator in docstring
```suggestion
        """Extracts all paths from the collector :return: Yields paths with size."""
```